packages/hackmode/lib/p2p-backend-hackmode-peer.js

```diff
@@ -328,7 +339,12 @@ class P2pBackendHackmodePeer extends AsyncClass {
   /**
    */
   async _removeConnection (cId) {
-    throw new Error('unimplemented')
+    for (let [peerAddress, cId] of this._conByPeerAddress) {
+      if (cId === cId) {
```

Comment:
@neonphog `cId === cId` huh?

Reply:
hah! yeah, that's no good. need to change the binding in the for loop, otherwise it'll delete all the connections!